llvm/unittests/CodeGen/SelectionDAGPatternMatchTest.cpp

```diff
@@ -603,3 +603,18 @@ TEST_F(SelectionDAGPatternMatchTest, matchAdvancedProperties) {
   EXPECT_TRUE(sd_match(Add, DAG.get(),
                        m_LegalOp(m_IntegerVT(m_Add(m_Value(), m_Value())))));
 }
+
+TEST_F(SelectionDAGPatternMatchTest, matchUndefined) {
+  auto BoolVT = EVT::getIntegerVT(Context, 1);
```

Comment:
nit: could you put tests in matchConstants instead?

Reply:
Done